sys-fs/compsize/compsize-1.1.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit flag-o-matic
+DESCRIPTION="Utility to find btrfs compression ratio"
+HOMEPAGE="https://github.com/kilobyte/compsize"
+
+if [[ ${PV} = 9999 ]] ; then
+	inherit git-r3
+	KEYWORDS=""
```

Comment:
Remove the two empty vars. Otherwise `ekeyword` used by arch testers will add keywords here.